modules/network-observability-roles-create.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-roles-create_{context}"]
+
+= Create roles for authentication and authorization
+Specify authentication and authorization configurations by defining `ClusterRole` and `ClusterRoleBinding`. You can create a YAML to define these roles.
+.Procedure
+
+. Using the web console, navigate to the Imoport icon *+*, in the top right corner. 
```

Comment:
Import icon? Typically, the prepositional phrase (top right corner) is omitted in UI text since it might get moved around.

Reply:
Its a plus-sign icon in the console that is for importing yaml files. I can't find the icon in the images directory and tbh didn't feel like/didn't have time to haggle people to add it. Its a plus sign nmw 🤷‍♀️